UI/MapboxNavigationUI/MGLMapView.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+import Mapbox
+import MapboxDirections
+import MapboxGeocoder
+
+let sourceIdentifier = "routeSourceID"
+let routeLayerIdentifier = "routeLayerIdentifier"
+let routeLayerCasingIdentifier = "routeLayerCasingIdentifier"
+
+extension MGLMapView {
+    public func show(_ placemark: GeocodedPlacemark) {
+        if let bounds = placemark.coordinateBounds {
+            setVisibleCoordinateBounds(bounds, animated: true)
+        } else {
+            setCenter(placemark.location.coordinate, zoomLevel: placemark.preferredZoomLevel, animated: true)
+        }
+    }
+    
+    public func annotate(_ routes: [Route], clearMap: Bool) {
+        
+        // We don't support alternative routes at this point
```

Comment:
This is a notable limitation, given that the developer may choose a profile that supports alternative routes; that is, any profile other than `MBDirectionsProfileIdentifierAutomobileAvoidingTraffic`. Alternative routes will require much more than UI, so let’s ticket that out as a separate effort.